src/core/ext/filters/client_channel/retry_throttle.cc

```diff
@@ -49,11 +49,11 @@ ServerRetryThrottleData::ServerRetryThrottleData(
   // we will start out doing the same thing on the new one.
   if (old_throttle_data != nullptr) {
     double token_fraction =
-        static_cast<intptr_t>(
+        static_cast<double>(
```

Comment:
I can change this one.

Reply:
Hmm, okay.  I somehow didn't realize that the compiler handled these conversions by converting the operands *before* performing the operation (although now that I think about it, it would be surprising if it worked any other way).

In this particular case, though, there's an extra complication in that the value is actually stored in a `gpr_atm`, and the existing cast actually informs future readers of this code about what value type we're actually storing in it, which is misleading with this change.  So let's add a comment here explaining that the value stored in `milli_tokens_` is actually an `intptr_t`, not a `double`, but that we're converting it to double here to avoid the compiler warning.